chassis_encap.go

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Please follow the convention to use table name as file name. i.e. encap.go

Reply:
We follow same for other components with what its tied to e.g. logical_router_static_route.go, etc. So kept it. Is it ok?